### ISSUE TITLE
Crash when parsing HTML string with the value of " "

### ISSUE THREAD

**Issue by ivan-magda** [tid-kijyun/Kanna#242]
### Description:

With the latest release 5.2.3, I discovered crashes when parsing HTML:
```
Fatal error: Unexpectedly found nil while implicitly unwrapping an 
Optional value: file Kanna/libxmlHTMLNode.swift, line 144
```

Code to reproduce:
```swift
try? Kanna.HTML(html: " ", encoding: String.Encoding.utf8)
```

### Installation method:
- [ ] Carthage
- [x] CocoaPods
- [ ] Swift Package Manager
- [ ] Manually
- [ ] other: ()

### Kanna version (or commit hash):
5.2.3
### swift --version
5.3
### Xcode version (optional):
12.0.1

**Comment by Krish-sysadmin** [comment]
@ivan-magda I think I can fix this. It simply involves modifying the libxmlHTMLNode.swift file. Can this be assigned this to me?

**Comment by tid-kijyun** [comment]
I'm sorry I missed this report.
@Krish-sysadmin Please make a PR if possible.

Thanks

**Comment by Krish-sysadmin** [comment]
Ok @tid-kijyun . Will make it as soon as I am free. Thanks for providing such an amazing parser!

**Comment by Krish-sysadmin** [comment]
I submitted a PR @tid-kijyun

**Comment by tid-kijyun** [comment]
I've just released new version [5.2.6](https://github.com/tid-kijyun/Kanna/releases/tag/5.2.6). Please try it.
